service/worker/migration/workflow.go

```diff
@@ -284,7 +284,11 @@ func NamespaceHandoverWorkflow(ctx workflow.Context, params NamespaceHandoverPar
 		return err
 	}
 
+	metricsHandler := workflow.GetMetricsHandler(ctx).WithTags(
+		map[string]string{"migrate-namespace": params.Namespace, metrics.TargetClusterTagName: params.RemoteCluster})
```

Comment:
I think you can just hardcode "target_cluster" here, or even use "remote_cluster", no need to expose that tag name.

Reply:
The pattern employed is to keep the constant not exported and use an exported function that takes the value as a parameter and return the materialized map.